index.html

```diff
@@ -304,7 +302,7 @@ <h3>Navigation</h3>
                         </p>
                     </div>
                     <div class="col-12 col-md-6 copyright">
```

Comment:
can you add the link to privacay and term of use to or footer? so you people can access it without using the app

Reply:
Sure, that makes sense